src/index.js

```diff
@@ -23,8 +21,8 @@ function base (ALPHABET) {
   var FACTOR = Math.log(BASE) / Math.log(256) // log(BASE) / log(256), rounded up
   var iFACTOR = Math.log(256) / Math.log(BASE) // log(256) / log(BASE), rounded up
   function encode (source) {
-    if (Array.isArray(source) || source instanceof Uint8Array) { source = _Buffer.from(source) }
-    if (!_Buffer.isBuffer(source)) { throw new TypeError('Expected Buffer') }
+    if (Array.isArray(source)) { source = Uint8Array.from(source) }
```

Comment:
@achingbrain If source is an `ArrayBufferView` it will copy a memory, I think there some savings to be had in by avoiding that maybe

```suggestion
    if (source instanceof Uint8Array) {}
    else if (ArrayBuffer.isView(source)) { source = new Uint8Array(source.buffer, source.byteOffset, source.byteLength) }
    else if (Array.isArray(source)) { source = Uint8Array.from(source) }
```